README.md

```diff
@@ -2,11 +2,38 @@
 
 The Akeneo PIM documentation
 
-## Installation
+## Build the documentation
+
+### With Docker
+
+Install [Docker](https://docs.docker.com/engine/installation/).
+
+[optional] To update the branch list with the current pim-docs branches, use:
+
+```
+sed -i -e "s/^\(.*\)'versions': .*,\(.*\)$/\1'versions': ['$(git branch -l|grep -x "\(^[ *]\+[0-9]\+\.[0-9]\+.*\)\|\(^[ *]\+master\)" | cut -c 3- | sort -r | paste -sd " ")'],\2/" conf.py
```

Comment:
This is not true anymore. You can drop this.
Versions are done with this
https://github.com/akeneo/pim-docs/pull/1208/files#diff-c09a020a80f3e0d8babc053a99898133